nextflow_schema.json

```diff
@@ -587,7 +605,6 @@
                 "bt2n": {
                     "type": "integer",
                     "description": "Specify the -N parameter for bowtie2 (mismatches in seed). This will override defaults from alignmode/sensitivity.",
-                    "default": 0,
```

Comment:
Why has this been removed (and below)?

Reply:
I fear that was when updating with the schema builder...?

Reply:
Old version of tools i suppose (1.12.X?)